src/types.ts

```diff
@@ -1,10 +1,11 @@
 export interface PanelOptions {
   xAxis: XAxisOptions;
   yAxis: YAxisOptions;
-  yAxis2?: YAxisOptions;
+  yAxis2: YAxisOptions;
```

Comment:
Can you explain why this is no longer nullable?

Reply:
I'm not 100% sure why it was in the first place, but I can confirm that creating an empty panel will always initialize that property to an empty array.